nixos/modules/services/libeufin/module.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  pkgs,
+  config,
+  ...
+}:
+
+let
+  settingsFormat = pkgs.formats.ini { };
+  this = config.services.libeufin;
```

Comment:
Should be done in cfg and be rolled up

Reply:
What do you mean here exactly?

Reply:
Rolled up as in the commit should be merged with a previous one as it is a fixup commit.